### DISCUSSION TITLE
HDMI support

### DISCUSSION THREAD

**Discussion by RealBanlog** [brad-lin/FreePSXBoot#84] [Ideas]
Hello everyone, this is my first post.

I use a Bitfunx HDMI adapter (RGB signal), which works wonderfully. Unfortunately, there are problems when I start Unirom (screen stays black). I suspect that the resolution is no longer compatible from that point. Is it possible that the next update you choose a resolution that could then work with the Bitfunx? Thanks a lot!

**Comment by realJoshByrnes** [comment]
I have a cheap RCA to HDMI adapter, and don't notice any change. Are you seeing the loader screen? Have you tried it directly without the adapter?
